core/src/test/java/com/coderanch/blackjack/HandTest.java

```diff
@@ -209,7 +209,7 @@ public void bestScore_hasCorrectScore(BestScoreTestArgument scoreTest) {
      */
     @Theory
     @SuppressWarnings("checkstyle:methodname")
-    public void withAdditionalCard_hasCorrectScore(Hand hand, Card card) {
+    public void withAdditionalCard_hasCorrectScore(@FromDataPoints("Add card hands") Hand hand, Card card) {
         if (card.rank() == Rank.ACE) {
             return;
         }
```

Comment:
Sorry, I should have caught this earlier.

When testing with Theories, guard against arguments you're not interested in using an assumption, not an if-statement:

```suggestion
        assumeThat(card.rank(), is(not(Rank.ACE)));
```

The practical difference between the two is that when you pass an Ace, an if-statement will report the test as having succeeded, while the assumption will report the test as having been skipped.